parsl/data_provider/data_manager.py

```diff
@@ -156,22 +122,8 @@ def stage_out(self, file, executor):
         elif file.scheme == 'ftp':
             raise Exception('FTP file staging out is not supported')
         elif file.scheme == 'globus':
-            globus_ep = self._get_globus_endpoint(executor)
-            stage_out_app = self._globus_stage_out_app()
-            return stage_out_app(globus_ep, inputs=[file])
+            globus_scheme = _get_globus_scheme(self.dfk, executor)
```

Comment:
In the case of multi-site with globus enabled sites, we may not know what the stage_out destination is. In that situation, we just need to pass a pointer to where the file is and have the next stage_in handle the actual transfer.